File: boilerplate/backend/python-flask/config/emailpassword.py
from supertokens_python.recipe import emailpassword, session
from supertokens_python import (
    InputAppInfo,
    SupertokensConfig,
)

# this is the location of the SuperTokens core.
supertokens_config=SupertokensConfig(connection_uri="https://try.supertokens.io")

app_info=InputAppInfo(
    app_name="Supertokens",
    api_domain="http://localhost:3001",
    website_domain="http://localhost:3000",
)

framework="flask"

# recipeList contains all the modules that you want to
# use from SuperTokens. See the full list here: https://supertokens.com/docs/guides
recipe_list=[
    session.init(),
    emailpassword.init()
]

File: boilerplate/backend/nest/config/emailpassword.ts
import EmailPassword from "supertokens-node/recipe/emailpassword";
import Session from "supertokens-node/recipe/session";

export const appInfo = {
    // Learn more about this on https://supertokens.com/docs/thirdpartypasswordless/appinfo
    appName: "ST",
    apiDomain: "http://localhost:3001",
    websiteDomain: "http://localhost:3000",
    apiBasePath: "/auth",
    websiteBasePath: "/auth",
}

export const connectionUri = "https://try.supertokens.com";

export const recipeList = [
    EmailPassword.init(),
    Session.init(),
];